__tests__/app.test.js

```diff
@@ -215,6 +215,42 @@ describe("GET", () => {
       });
   });
 
+  test("/api/users/ 200: responds with all the users in the users table (using SQL of course).", () => {
+    request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          Users: [
```

Comment:
would be better to write assertions within a for each rather than test against this big output.